lib/src/functions/parse/parse.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:convert';
+
+  /// Makes cmd arguments JSON parsable
+  //  Turns [{asdf:asdf}] -> [{"asdf":"asdf"}]
+parseArgsJson(str) {
+    String parse0 = str;
+    String parse1 = parse0.replaceAll("\'", ('\"'));
+    // String parse2 = parse1.replaceAll("{", ('{\"'));
+    // String parse3 = parse2.replaceAll("}", ('\"}'));
+    Map parse4 = json.decode(parse1);
+    Map parse5 = Map<String, String>();
+    parse4.forEach((k, v) => parse5[k is String? k.trim() : k.toString()] = v is String ? v.trim() : v.toString());
+    
+    // print(parse5);
+    return parse5;
```

Comment:
In this case the output will always be a string, so does that mean we should cast it to its wanted data type after doing this conversion? Ideally we want to tell the user if their input is of the right type or not immediately

Reply:
Yeah this is just temporary because I dont know what entry fields and datatypes we are taking in right now